mobile_app_connector/models/wordpress_post.py

```diff
@@ -92,10 +93,17 @@ def fetch_posts(self, post_type):
         category_obj = self.env['wp.post.category']
         found_ids = []
         try:
+            h = HTMLParser()
             with wp_requests.Session() as requests:
                 for lang in self._supported_langs():
                     params['lang'] = lang.code[:2]
                     wp_posts = requests.get(wp_api_url, params=params).json()
+                    for post in wp_posts:
```

Comment:
Why adding this loop and not just changing at line `162`?
```
 'name': h.unescape(post_data['title']['rendered']),
```

Reply:
Since the field "excerpt" is not used, we can definitely do as you propose.